### PR TITLE
bumping version of the opfcli binary to v0.3.0

### PR DESCRIPTION
**Pull request by Gregory-Pereira**
## Related Issues and Dependencies

Related to: https://github.com/operate-first/opfcli/pull/44
Related to [thoth-station/thoth-application issue 2111](https://github.com/thoth-station/thoth-application/issues/2111)

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Bumping version of opfcli to new version `v0.3.0` to include new `onboard` command.

Additional change has been included to use the `https` protocol instead of the unencrypted `git` protocol.

/cc @HumairAK

### CODE REVIEW

**Comment by Gregory-Pereira** [comment]
not sure why this failed.
/retest

**Comment by HumairAK** [comment]
stop fighting me sesheta
/test all

**Comment by HumairAK** [comment]
@harshad16 how can I attempt this build check again?

**Comment by Gregory-Pereira** [comment]
@HumairAK I don't think this is actually a test, my command above lists all tests and its not showing up. Also the pipeline-run succeeds but the build-check condition fails. Maybe it needs to be added to the `.aicoe-ci.yaml` check value?

**Comment by larsks** [comment]
The commit "[bumping version of the opfcli binary to v0.3.0](https://github.com/operate-first/toolbox/pull/66/commits/c472398c5e7742596e0f1016f41444a983a66b71)" seems to be doing a lot more than just bumping the version of opfcli. Maybe it should be split?

**Comment by Gregory-Pereira** [comment]
> The commit "[bumping version of the opfcli binary to v0.3.0](https://github.com/operate-first/toolbox/pull/66/commits/c472398c5e7742596e0f1016f41444a983a66b71)" seems to be doing a lot more than just bumping the version of opfcli. Maybe it should be split?

Great point, I am on it. I will remove the version bump with the changes to the underlying images that make it buildable / runable. 
/hold

File: Dockerfile [hunk 1]
```diff
@@ -33,11 +33,16 @@ LABEL maintainer="Operate First" \
     version.ksops="${KSOPS_VERSION}" \
     version.sops="${SOPS_VERSION}"
 
+ADD ./labels.yaml /etc/config/labels.yaml
+ADD ./labels.yaml /gen-source/labels.yaml
+
+RUN ls /gen-source/
```
Comment (harshad16) [inline Dockerfile:39]:
```suggestion
RUN \
   curl -o /etc/config/labels.yaml https://raw.githubusercontent.com/operate-first/common/main/labels.yaml && \
   cp /etc/config/labels.yaml  /gen-source/labels.yaml
```

we can do this way. move this run command is already present run if this not needed before COPY.

**Review by harshad16 — CHANGES_REQUESTED** [review]
let not copy the labels as it make maintenance tougher.
here a suggestion that could work.

**Comment by Gregory-Pereira** [comment]
Commits have been refactored and tests pass, ready for review @HumairAK @harshad16

**Review by harshad16 — APPROVED** [review]
/lgtm
thanks :100:

**Pull request merged**